red.py

```diff
@@ -375,6 +375,10 @@ def initialize(bot_class=Bot, formatter_class=Formatter):
         else:
             bot.logger.exception(type(error).__name__, exc_info=error)
 
+    @bot.event
+    async def on_error(error, ctx):
+        bot.logger.exception(type(error).__name__, exc_info=error)
```

Comment:
Needs to be changed to `bot.logger.error("Error in event: {}".format(event_name), exc_info=sys.exc_info())